runner.go

```
@@ -46,14 +49,30 @@ func (r *Runner) RunWithoutErrors(promptContext prompt.PromptContext) int {
 	return r.exitStatus()
 }
 
+func getCommitCount() uint64 {
+	out, err := exec.Command("git", "rev-list", "--all", "--count").CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+	result := strings.Split(string(out), "\n")[0]
+	count, _ := strconv.ParseUint(result, 10, 64)
```

Why ignore the error ?